stdlib/public/core/StringBridge.swift

```diff
@@ -134,10 +134,15 @@ internal enum _KnownCocoaString {
   case storage
   case shared
   case cocoa
+  case constantCocoa
 #if !(arch(i386) || arch(arm))
   case tagged
 #endif
   
+  //TODO: figure out a nicer way to get the constant NSString class
+  static let constantClass = _unsafeAddressOfCocoaStringClass(
+    ""._bridgeToObjectiveCImpl())
```

Comment:
Does this actually give you an NSConstantString, as opposed to just a Swift string class of some kind?

Reply:
It does! I realize this is evil. I could also add a shim for objc_lookUpClass(), but I have a vague concern that this might be safer than hardcoding _NSCFString.

Reply:
I was going to suggest putting a string constant in SwiftShims instead.